src/js/story-factory.js

```diff
@@ -0,0 +1,39 @@
+const Story = (title) => {
+  let chapters = [];
+  let completed = false;
+
+  const addChapters = (...chapterData) => {
+    chapterData.forEach((data) => {
+      chapters.push(data);
+    });
+  };
+
+  const findFirstChapter = () => {
+    return chapters.find(
+      (firstChapter) => firstChapter.getChapterNumber() === 1
+    );
+  };
+
+  const findNextChapter = (currentChapter) => {
+    const chapterNumber = currentChapter.getChapterNumber();
+
+    return chapters.find(
+      (nextChapter) => nextChapter.getChapterNumber() === chapterNumber + 1
+    );
+  };
+
```

Comment:
Everything nice, in addition one function to get the quizzes for the final quiz page would be useful: 

```js 
// returns an array with quizzes of every second chapter
const getFinalQuizzes = () => {
  return chapters.reduce((acc, cur, i) => {
    if (i % 2 === 0) {
     acc.push(cur.quiz)
    }
    return acc
  }, [])
}
```

Reply:
So our final quiz will be just a mix of half of the quizzes from chapters?

Reply:
I am open also for something different. What do you have in mind?

Reply:
I was just asking to be sure, for now i have nothing in mind. Will add your suggestion to the factory. I will close this PR and add a new one because i messed up with branches.